lib/trento/hosts/host.ex

```diff
@@ -259,10 +260,15 @@ defmodule Trento.Hosts.Host do
         total_memory_mb: total_memory_mb,
         socket_count: socket_count,
         os_version: os_version,
-        fully_qualified_domain_name: fully_qualified_domain_name,
+        fully_qualified_domain_name: new_fully_qualified_domain_name,
         installation_source: installation_source
       }
-    ]
+    ] ++
+      maybe_emit_software_updates_discovery_events(
+        host_id,
+        current_fully_qualified_domain_name,
```

Comment:
Shouldn't this 2nd argument be `nil` here?
I mean, I understood that in this circunstance we don't want to emit the `Cleared` event, but if the new fqdn is `nil`, and we had a value before (current), it would be emitted, or not?

Reply:
yep, I guess I missed that test.
Here we go https://github.com/trento-project/web/pull/2518
Thanks!